tensorflow_addons/image/testing_imageops.ipynb

```diff
@@ -0,0 +1,728 @@
+{
```

Comment:
<p>We try to stay away from using anything <code>tensorflow.python</code> as these are private API calls. Could You use PIL instead since it's already imported?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/368/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>